core/miner/main/main.cpp

```diff
@@ -96,6 +100,24 @@ namespace fc {
     auto join(const std::string &path) const {
       return (repo_path / path).string();
     }
+
+    const std::string &localIp() const {
```

Comment:
move inline function to core/common header (reuse in node main)

Reply:
Done